plugins/backend/decsync/decsyncUtils.vala

```diff
@@ -43,23 +43,27 @@ public void setDecsyncDir(string decsyncDir)
 	Utils.gsettingWriteString(m_settings, "decsync-dir", decsyncDir);
 }
 
-public Feed? downloadFeed(Soup.Session session, string xmlURL, string feedID, Gee.List<string> catIDs, out string errmsg)
+public Feed? downloadFeed(Soup.Session session, string feed_url, string feedID, Gee.List<string> catIDs, out string errmsg)
 {
 	errmsg = "";
+	string error_prefix = _(@"Failed to add feed $feed_url\n");
```

Comment:
@bilelmoussaoui Is this the right way to make strings translatable? Do I need to do anything else, and is it ok that it has a variable in it?

Reply:
You should avoid having variables there and use printf, otherwise translators might translate the variable name too.

Reply:
Thanks, I switched this so the translatable strings don't have variables in them.